awx/main/tasks.py

```diff
@@ -1232,6 +1238,9 @@ def run(self, pk, **kwargs):
                 status = 'failed'
                 extra_update_fields['job_explanation'] = self.instance.job_explanation
 
+        except InvalidVenvError as e:
+            tb = e.message
+            logger.exception(tb)
```

Comment:
I don't love the logger message because it repeats the user-facing message without adding much useful context for debugging, and it's not really correct use of `logger.exception`, it'll print things twice weirdly. I would use a pattern like `logger.exception('{} had an invalid venv'.format(self.instance.log_format))` or `logger.error('{} did not start because its custom venv is missing'.format(self.instance.log_format))`. I would prefer the latter.